projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -136,6 +136,14 @@ export class TippyDirective implements OnChanges, AfterViewInit, OnDestroy, OnIn
     }
 
     this.setProps({ ...this.props, ...props });
+
+    if (isChanged<NgChanges<TippyDirective>>('content', changes) && !changes.content.isFirstChange()) {
+      if (!changes.content.previousValue && changes.content.currentValue) {
+        this.initInstanceCreation();
```

Comment:
There is a situation where we might subscribe several times to the events.
Let's say I'm starting with no content, then adding content. I'm subscribed twice no? 
I think the creation trigger should be uncoupled from the instance existence. If we already decided to create an instance, we should always create it here and not resubscribe to the events.